jdk_8_maven/cs/rest/original/swagger-petstore/src/main/java/io/swagger/petstore/Main.java

```diff
@@ -0,0 +1,100 @@
+//MODIFIED: This code is a simple Java application that starts an embedded Tomcat server to serve the Swagger Petstore application.
+
+package io.swagger.petstore;
+
+import org.apache.catalina.startup.Tomcat;
+
+import java.io.File;
+import java.net.URL;
+import java.nio.file.Files;
+
+public class Main {
+    Tomcat tomcat;
+
+    public void startServer(int port) throws Exception {
```

Comment:
@omursahin  hi. this feels bit weird. we would need to discuss it. you should be able to run things directly, without need to write tmp files on filesystem